src/test/java/com/ge/predix/uaa/token/lib/FastTokenServiceTest.java

```diff
@@ -48,15 +48,26 @@
 
     private static final String TOKEN_KEY_URL = "https://localhost:8080/uaa/token_key";
 
-    private final FastTokenServices services = new FastTokenServices();
+    private final FastTokenServices services;
 
     private final Map<String, Object> body = new HashMap<>();
 
     public FastTokenServiceTest() throws Exception {
+        this.services = new FastTokenServices();
+        this.services.setRestTemplate(mockRestTemplate());
+        this.services.setTrustedIssuers(trustedIssuers());
+    }
+
+    private List<String> trustedIssuers() {
+        List<String> trustedIssuers = new ArrayList<>();
+        trustedIssuers.add(TOKEN_ISSUER_ID);
+        return trustedIssuers;
+    }
 
+    private RestTemplate mockRestTemplate() {
         this.body.put(Claims.CLIENT_ID, "remote");
         this.body.put(Claims.USER_NAME, "olds");
-        this.body.put(Claims.EMAIL, "olds@vmware.com");
+        this.body.put(Claims.EMAIL, "olds@example.com");
         this.body.put(Claims.ISS, TOKEN_ISSUER_ID);
         this.body.put(Claims.USER_ID, "HDGFJSHGDF");
 
```

Comment:
The following tests might be failing because of the expired token and not because of the "untrusted issuer". 

testLoadAuthenticationWithOtherIssuerId
testLoadAuthenticationWithUnstrustedIssuerId

We should mock a token that is current:
this.testTokenUtil.mockAccessToken(60);